templates/saves.ejs

```diff
@@ -1,26 +1,42 @@
-<div class="response">
-    <button
-        hx-get="/"
-        hx-target="#content"
-        hx-swap="innerHTML"
-        hx-push-url="true"
-    > 
-        Return to game
-    </button>
+<script>
+    function renameLocalStorageSave(save_key, save_name_id) {
```

Comment:
I am leaving the method here for now. 
Based on what Claud mentioned, I feel like raising a new refactoring PR that splits the utils file would be the way to go.